runtime/doc/api.txt

```diff
@@ -700,6 +700,20 @@ nvim_chan_send({chan}, {data})                              *nvim_chan_send()*
       • {chan}  id of the channel
       • {data}  data to write. 8-bit clean: can contain NUL bytes.
 
+nvim_complete_set_info({index}, {text})             *nvim_complete_set_info()*
```

Comment:
Does vim have a vimscript function for this, or why is this needed in Nvim API?

Note about naming: we have `nvim_select_popupmenu_item`, should this be named `nvim_set_popupmenu_item`? Or why not?

Shoud also have a `{opts}` arg for future expansion, and should return `Dict` (even if it's currently empty).

Reply:
No. Vim creates a hidden info popup window by setting `popuphidden` in `completepopup` when the popupmenu is activated. When the info text is obtained asynchronously, modify popuphidden to popup to display the floating window.  is always a floating wp created even get info failed.

api  is used here. it is invoked to set info only when asynchronously obtaining info is successful, and then a floating window is created and displayed only when the index is select index.

Reply:
> nvim_set_popupmenu_item

unclear. The purpose of the API is to set the info text for a popupmenu item index

Reply:
emm even weirder. main purpose is to set the info field for the candidate.

Reply:
I mean, `nvim_complete_set_info` doesn't set anything...

Reply:
?? it set text to cp_text[CPT_INFO]